src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage.jsx

```diff
@@ -63,30 +62,44 @@ export default function VAPServiceEditModalErrorMessage({
 
     default:
       content = (
-        <p>
+        <p id="error-message-details">
           We’re sorry. We can’t update your information right now. We’re working
           to fix this problem. Please check back later.
         </p>
       );
   }
 
   return (
-    <AlertBox
-      backgroundOnly
-      status="error"
-      className="va-profile-alert vads-u-margin-y--1"
-    >
-      <div className="vads-u-display--flex">
-        <i
-          className="fas fa-info-circle vads-u-font-size--md vads-u-color--black vads-u-margin-right--2 vads-u-padding-top--0p5"
-          aria-hidden="true"
-          role="img"
-        />
-        <span className="sr-only">Alert: </span>
-        <div role="alert" aria-live="polite">
-          {content}
+    <>
+      <va-alert
+        background-only
+        close-btn-aria-label="Close notification"
+        status="error"
+        visible
+      >
+        <div className="vads-u-display--flex vads-u-align-items--baseline vads-u-margin-left--neg1p5 vads-u-margin-y--neg1p5">
+          <i
```

Comment:
Does the icon have to be black here? It seems like it would make more sense to add `show-icon` to the `va-alert` and just use the red circle icon